### ISSUE TITLE
Group label displaying main icon under the others

### ISSUE THREAD

**Issue by ryuji-terix** [Alexays/Waybar#3285]
Using the default configuration I can't get the group to display correctly (even trying with all possible "direction"s)
The problem is that the "power off" button stays "halfed" even when collapsed like in a vertical direction

Collapsed:
![main_icon](https://github.com/Alexays/Waybar/assets/64895799/366355f1-730d-4f71-8a06-db9d37ce11d6)

Hovered:
![group_expanded](https://github.com/Alexays/Waybar/assets/64895799/f1ea3d19-83d8-4fa5-98af-3ec6575164d1)

Configuration:
![20240520_20h05m15s_grim](https://github.com/Alexays/Waybar/assets/64895799/57c6a0ef-eba0-4d21-8420-e894b197aea3)

**Comment by ryuji-terix** [comment]
I have copy-pasted the other example code (the one for the hardware modules) and it worked as normal, so I tried slowly moving every configuration from the broken snippet (image in description) to the second example and now it works, the only thing I left unchanged was the name.

TLDR `group/power` must be some sort of reserved name for something (or `#power`) that messes up with a group display
